### DISCUSSION TITLE
Highlighter not filling spaces between words after OCRmyPDF

### DISCUSSION THREAD

**Discussion by Nicolai9852** [ocrmypdf/OCRmyPDF#1285] [Q&A] [answered]
While I have been using OCRmyPDF for a little while, there is still a issue, I havn't been able to figure out.
After turning af PDF into one with an OCR layer, the highlighter tool in Acrobat doesn't fill the gaps between words. Has anyone experienced the same or knows a solution?
![image](https://github.com/ocrmypdf/OCRmyPDF/assets/24588044/57317f5e-6f6e-450f-a30c-2316feb3ce28)

**Comment by jbarlow83** [comment, marked as answer]
Likely you're using a version before 16 when text placement was improved.

**Reply by Nicolai9852** [reply]
I'll be honest, I hadn't checked for updates, since I wasn't aware of it. The new update worked like a charm.
Thank you @jbarlow83 very kindly for maintaining the project. I really like it and have used it a lot - and still does.
